src/js/utils/DOMUtils.js

```diff
@@ -28,21 +29,19 @@ var DOMUtils = {
 
     var computeInnerBound = function (acc, key) {
       var val = parseInt(compstyle[key], 10);
-      if (_.isNaN(val)) {
+      if (Number.isNaN(val)) {
```

Comment:
It seems that the `parseInt(value, 10);` has got us covered:
![](http://cl.ly/1w0L0637090W/Image%202016-05-03%20at%2014.26.23.png)